src/Provider/WFSProvider.js

```diff
@@ -89,7 +84,7 @@ function getFeatures(crs, tile, layer) {
                     throw err;
                 }
             })
-        .then(feature => assignLayer(layer.convert(feature), layer));
+        .then(feature => Promise.resolve(assignLayer(layer.convert(feature), layer)));
```

Comment:
I don't think this change is necessary. Otherwise the rest looks good.

Reply:
It is not indeed, I pushed the change !